modules/openxr/extensions/openxr_fb_foveation_extension.cpp

```diff
@@ -56,9 +56,12 @@ OpenXRFBFoveationExtension::OpenXRFBFoveationExtension(const String &p_rendering
 
 	if (rendering_driver == "opengl3") {
 		swapchain_create_info_foveation_fb.flags = XR_SWAPCHAIN_CREATE_FOVEATION_SCALED_BIN_BIT_FB;
-	} else if (rendering_driver == "vulkan") {
+	}
```

Comment:
Shouldn't this also check `XR_USE_GRAPHICS_API_OPENGL`?

Reply:
It doesn't need it to compile, because the extension always has the `XR_SWAPCHAIN_CREATE_FOVEATION_SCALED_BIN_BIT_FB` constant regardless. But, yeah, we could avoid the string comparison if OpenGL isn't enabled at all using some `#ifdef`'s. Let me see how that looks..

Reply:
So, using `#ifdef`s for OpenGL here isn't a complete win, because without even more ugly `#ifdef`s, we end up losing the `else`, so when both OpenGL and Vulkan are enabled, it does two string comparisons instead of one. But this code only runs once at startup, so it's fine.

In fact, we could probably just not have `#ifdef`s on this code at all? These two constants are always available regardless of which graphics APIs are enabled. I think I'm going to go that way in my next push, but I'd be happy to switch to the other if that was preferred.